core-web/libs/block-editor/src/lib/shared/services/ai-content/ai-content.service.ts

```diff
@@ -13,9 +13,22 @@ interface OpenAIResponse {
 
 @Injectable()
```

Comment:
Keep this inside the lib block editor/

Reply:
This is part of the block editor- lives under libs/editor/lib/shared you mean on this or something else?